lua/nvim-lsp-ts-utils/define-commands.lua

```diff
@@ -12,6 +12,10 @@ local define_commands = function()
         buf_command("TSLspRenameFile", "rename_file()")
         buf_command("TSLspFixCurrent", "fix_current()")
         buf_command("TSLspImportAll", "import_all()")
+        -- Inlay hints
+        buf_command("TSInlayHints", "inlay_hints()")
```

Comment:
Could we change the prefix from `TS` to `TSLsp` for consistency (and to prevent confusion with Treesitter)?